lib/config/configuration.go

```diff
@@ -634,7 +634,7 @@ func ApplyFileConfig(fc *FileConfig, cfg *servicecfg.Config) error {
 			l.MaxConnections = fc.Limits.MaxConnections
 		}
 		if fc.Limits.MaxUsers > 0 {
-			l.MaxNumberOfUsers = fc.Limits.MaxUsers
+			log.Warn("Configuration option max_users is no longer supported")
```

Comment:
It should be obvious, but can we include the full configuration path?
`teleport.connection_limits.max_users`